docs/site/todo-tutorial-putting-it-together.md

```diff
@@ -47,7 +47,7 @@ Here are some requests you can try:
 - `GET /todos/{id}` using the ID you received from your `POST`, and see if you
   get your Todo object back.
 - `PATCH /todos/{id}` using the same ID, with a body of
-  `{ "desc": "need milk for cereal" }`
+  `{ "title": "get the milk", "desc": "need milk for cereal" }`
```

Comment:
Is this change needed because `title` is a required property and we don't support partial updates (real patches) yet because of https://github.com/strongloop/loopback-next/issues/1722?

Can we add a short explanation of this problem and point readers to #1722 please?

Reply:
Will do, thanks!